dom.bs

```diff
@@ -1651,6 +1651,24 @@ them. For instance, if the operation has already completed.
  <li><p>[=Fire an event=] named <code>abort</code> at <var>signal</var>.
 </ol>
 
+<p>A <var>followingSignal</var> (an {{AbortSignal}}) <dfn export for=AbortSignal>follows</dfn> a
+<var>parentSignal</var> (an {{AbortSignal}}) by running these steps:
```

Comment:
Nit: I think I'd rather make the algorithm have a name that isn't an adjective, but instead a verb. E.g. "A followingSignal is made to follow a parentSignal by..."

Reply:
Would the usage remain "signalA follows signalB" or should that change to "Make signalB follow signalB".

I struggled to find wording around the usage 😕.

Reply:
Probably "make signalB follow signalA". I guess I'd do `lt="follow"` then to make things easier.